core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
     userPhoto: Painter,
     timeAgo: String,
     messagePost: String,
-    sizeWidth: Float  = 1F,
+    sizeWidth: Float = 1F,
     onCommentClick: () -> Unit,
     onFavoriteClick: () -> Unit
-){
+) {
     Card(
         modifier = modifier
             .fillMaxWidth(sizeWidth)
             .padding(15.dp),
         colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primaryContainer),
         border = BorderStroke(1.dp, MaterialTheme.colorScheme.onPrimary),
         elevation = CardDefaults.cardElevation(1.dp),
-    ){
+    ) {
         Column(
             modifier = Modifier
                 .fillMaxWidth()
                 .padding(15.dp, 15.dp, 15.dp, 5.dp)
-        ){
+        ) {
             Row {
                 Column {
-                    Box(modifier = Modifier
-                        .fillMaxWidth(0.20f)
-                        .padding(0.dp)) {
+                    Box(
```

Comment:
you can remove this box and just have the image and the column in this row.
In the column you can add in modifier the property padding start to have separation between the image and all the texts.